### ISSUE TITLE
Silent's empty cells

### ISSUE THREAD

**Issue by WintersPride** [davqvist/Q-Tech#10]
The cells seem to bug out when being pulled out/put into machines from multiple origins. IE from the bottom left and top right, after pulling them out they return to original positions or drop on the ground when you exit the machine interface.

**Comment by davqvist** [comment]
First up I assume you mean canisters. Second I don't understand your setup, maybe attach a screenshot? Third, I tried it and couldn't replicate, so please provide more info so that I can replicate.

**Comment by WintersPride** [comment]
Yes I was talking about the canisters, used to fluid cells naming convention. I tried to upload a video but github doesn't support .mp4

**Comment by davqvist** [comment]
What do you mean with bottom left then? The gui or the machine itself? Can you describe exactly the steps to replicate it?
